learn/arazzo/documenting-multiple-apis-using-arazzo.md

```diff
@@ -0,0 +1,766 @@
+        successCriteria:
+          - condition: $statusCode == 200
+```
+
+These examples demonstrate how Arazzo can be used to document and implement both technical patterns and business processes. Each example includes:
+
+1. Clear workflow structure and dependencies
+2. Error handling and recovery strategies
+3. Data flow between steps
+4. Input validation and output mapping
+5. Success criteria and failure handling
+6. Real-world implementation patterns
+
+### API contract testing
+
+There's a vast difference between checking that individual components meet specifications (like testing that a door closes properly) and verifying that the entire building functions as designed. Arazzo enables this holistic testing approach for your APIs. For more details on testing, see [Testing Arazzo Workflows](./testing-arazzo-workflows.md).
+
+Redocly's internal tool (Respect) has long emphasized the importance of testing API workflows rather than just individual API calls. With Arazzo, this approach becomes standardized and accessible to all API producers.
+
+Contract testing with Arazzo allows you to:
+
+* Validate that entire user journeys work as expected  
+* Test interactions between multiple microservices or APIs  
+* Ensure that data flows correctly from one step to the next  
+* Detect breaking changes that might not be apparent from testing individual endpoints
+
+### End-to-end API testing
+
+End-to-end testing with Arazzo goes beyond contract testing by validating complete user journeys in real-world scenarios. This approach is like conducting a full building simulation with actual users rather than just verifying plans meet code requirements.
+
+With Arazzo-based end-to-end testing, you can:
+
+* Automate testing of entire workflows in CI/CD pipelines  
+* Validate business-critical journeys across multiple systems  
+* Test with real or mocked dependencies  
+* Create living documentation that proves your APIs work as described
+
+For the latest updates on our Arazzo support, check out our [Arazzo roadmap blog post](https://redocly.com/blog/arazzo-roadmap).
+
+## Get started with Arazzo
+
+Ready to transform your API documentation from static blueprints to complete architectural plans with circulation patterns? Here's how to get started:
+
+1. **Learn from real examples**: Review the examples in this article, including the [Healthcare Patient Onboarding](#healthcare-patient-onboarding) workflow and [Technical Pattern-Based Examples](#technical-pattern-based-examples) sections above.
+2. **Create your first workflow**: Start with a simple, common use case that involves multiple API calls. Check out [Arazzo Basics](./arazzo-basics.md) for a gentle introduction.
+3. **Validate your workflow**: Use the Redocly CLI to lint and validate your Arazzo files. Learn more about [Success Criteria and Failure Handling](./success-criteria-and-failure-handling.md):
+```sh
+npx @redocly/cli@latest lint YOUR_ARAZZO_FILE
+```
```

Comment:
I guess these examples are supposed to be part of the numbered lists? I think this should be indented 2 spaces (same with bullet 5 code sample).